src/main/java/org/glassfish/concurro/cdi/lock/Lock.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2025 OmniFish and/or its affiliates.
```

Comment:
https://github.com/eclipse-ee4j/glassfish-concurro/pull/105#discussion_r1932007400

Reply:
It is done!